adam/ontology/integrated_learner_experiement_ontology.py

```diff
@@ -116,12 +116,24 @@
 subtype(GLIM, INANIMATE_OBJECT)
 ZUP = OntologyNode(
     "zup",
-    [CAN_FILL_TEMPLATE_SLOT, PERSON_CAN_HAVE, ROLLABLE, RED, INTEGRATED_EXPERIMENT_PROP],
+    [
```

Comment:
Did black reformat this code? If its needed to pass the code-formatter now we can keep this change but otherwise I don't see a reason for this change to occur in a non-related ontology.

Reply:
It